rotating_proxies/__init__.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.3.1'
+__version__ = '0.3.2'
```

Comment:
Could you please remove version bump? I prefer to do it myself. Keeping a record in CHANGELOG would be nice, but instead of a version and date please use 'TBA' (meaning "to be announced).